### PR TITLE
Removed minikube references

### PR DESCRIPTION
**Pull request by andreas-kupries**
drop: remove minikube documentation and references to it.
tweak: some reformatting of overlong lines

### CODE REVIEW

File: docs/references/system_requirements/global.md [hunk 1]
```diff
@@ -11,7 +11,6 @@ For the Epinio server, and related deployments we recommend to consider the foll
 - Supported operating systems
   - Linux
     - K3s, K3d
```
Comment (thehejik) [inline docs/references/system_requirements/global.md:13]:
```suggestion
    - K3s, K3d, RKE2
```

Comment (andreas-kupries) [reply docs/references/system_requirements/global.md:13]: Hm. Maybe also Rancher Desktop, as is done in the Windows item just below.
RKE2 is full-blown server ...
That said I will make the names all links to the respective installation pages.

Comment (andreas-kupries) [reply docs/references/system_requirements/global.md]: Done with commit 75499a4b047ab70364b0ab5a0896642b4ea9c59b

Comment (thehejik) [reply docs/references/system_requirements/global.md]: Better, thanks

**Review by thehejik — APPROVED** [review]
LGTM

**Pull request merged**